lib/ansible/modules/system/user.py

```diff
@@ -935,6 +935,207 @@ def modify_user(self):
         return (rc, out, err)
 
 
+class DragonFlyUser(User):
```

Comment:
maybe just subclass freebsd one and then if there is a divergence you can override those methods

i would prefer that over soo much duplicate code